### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by chrisjansky** [nuxt/devtools#709] [Q&A]
I am using Nuxt 3.12.4 with Nuxt DevTools 1.3.9.

The opening DevTools screen says "updates available" but clicking it only opens the "Hi there, welcome to Nuxt DevTools!" screen and does nothing.

Is there a way to force downloading the latest DevTools version perhaps through a `npx nuxi` command?

Thanks.

**Comment by johnfisherman** [comment]
Following your lead, I did:

`npx nuxi upgrade`

and it updated everything for me. That worked well for me:

`Successfully upgraded Nuxt from 4.3.1 to 4.4.2`

DevTools included.

**Comment by chrisjansky** [comment]
Seeing I filed this issue almost 2 years ago, it is perfectly possible it has been fixed. Closing.
